src/query/execute_schema.c

```diff
@@ -10502,6 +10502,7 @@ do_change_att_schema_only (PARSER_CONTEXT * parser, DB_CTMPL * ctemplate, PT_NOD
       goto exit;
     }
 
+
```

Comment:
It is trivial.
Why was the empty line added?

Reply:
It's just a slip. Thanks.